cmd/endpoint.go

```diff
@@ -452,6 +473,26 @@ func (endpoints Endpoints) UpdateIsLocal(foundPrevLocal bool) error {
 					endpoints[i].Hostname(),
 				)
 
+				if orchestrated && hostResolveToLocalhost(endpoints[i]) {
```

Comment:
the cluster won't start if the endpoint which corresponds to the local machine that is running this code does not resolve to a non-loopback interface, is that right? 

🤞

Reply:
yes @vadmeste